draft-ietf-httpbis-http2bis.xml

```diff
@@ -538,9 +539,10 @@ HTTP Frame {
           processes as header compression and decompression.
         </t>
         <t>
-          Field blocks carry the compressed bytes of a field section, with header sections
-          also carrying control data associated with the message in the form of <xref target="PseudoHeaderFields">pseudo-header fields</xref> that use the same format as a
-          field line.
+          A field section consists of exactly one field block.  Each field block carries the
+          compressed bytes of a single field section.  Header sections also include control data
```

Comment:
Hmm, this section reads strangely to me. Field sections are made up of field blocks which carry the compressed bytes of field sections? This feels a little recursive.

Do we have a clearer way to express this idea?

Reply:
I gave it a try.  I had to submit a version, so I went with this:

>          Each field block carries all of the compressed field lines of a single field section.

Reply:
I think that would work much better.